scripts/mirror.js

```diff
@@ -574,6 +631,17 @@ const bumpVersion = (data, destination, source, originalData) => {
     newData = bumpFunction(originalData, data, source);
   }
 
+  if (targetVersion) {
+    if (
+      !(
+        isVersionAdded(newData, targetVersion) ||
+        isVersionRemoved(newData, targetVersion)
+      )
+    ) {
```

Comment:
Wondering whether this might be more readable:

```suggestion
    if (
        !isVersionAdded(newData, targetVersion) &&
        !isVersionRemoved(newData, targetVersion)
    ) {
```